python/tk_framework_desktopserver/process_manager.py

```diff
@@ -195,6 +196,8 @@ def execute_toolkit_command(self, pipeline_config_path, command, args):
 
             return (out, err, return_code)
         except Exception, e:
+            # call_cmd is not including sentitive information in the error message, so this won't
```

Comment:
This command feels unnecessary - the previous comments I felt were helpful, pointing out that we should be omitting something from the error message that otherwise would seem natural to include - thereby preventing future users to adding it (back) in and serve as an explanation as to why it isn't there. But this comment feels like we are sort of almost saying "there is nothing here to do so therefore we aren't doing anything" !